src/FSharp.Core/eventmodule.fsi

```diff
@@ -11,6 +11,8 @@ open Microsoft.FSharp.Control
 [<CompilationRepresentation(CompilationRepresentationFlags.ModuleSuffix)>]
 [<RequireQualifiedAccess>]
 module Event =
+    [<CompiledName("Create")>]
+    val create<'T> : unit -> ('T -> unit) * IEvent<'T>
```

Comment:
@vzarytovskii I wasn't sure about the additions to the signature files of F# Core.
As they were not part of F# Core, you could also opt to remove the implementation code.

Reply:
It's a strong hint about nobody caring I would say.

Reply:
We have spoken about it with Kevin and we propose to rather remove it.
It does not have documentation, usage samples etc. (and since I never worked with eventmodule at all, I would not feel confident adding sample of usage).

=> as a rule of thumb, we should not add new APIs as part of cleanup.
If a new API is desired, people can raise an issue and it can be added via a separate PR meeting standards for testing, documentation etc.

Reply:
That makes a lot of sense. I reverted all changes to FSharp.Core that adds anything to the surface area.